announcements/2020-Week-24.md

```diff
@@ -0,0 +1,29 @@
+# This Week in Releases (12.06.2020)
```

Comment:
```suggestion
# This Week in Releases (June 12, 2020)
```

DD.MM.YYYY is not a date format that makes sense to everybody internationally. Spelling out the month is really the only way you can make it pretty much fail safe.